translations/expansion_content.tex/de.po

```diff
@@ -161,6 +173,33 @@ msgid ""
 "\\end{minipage}\n"
 "\n"
 msgstr ""
+"\\begin{minipage}[t]{0.48\\textwidth}\n"
+"  \\centering\n"
+"  \\minipage[b]{0.5\\textwidth}\n"
+"    \\centering\n"
+"    \\includegraphics[width=0.75\\linewidth]{\\images/school_of_fire.png}\n"
+"    \\textit{\\textbf{\\textcolor{darkcandyapplered}{Feuermagie}}}\n"
```

Comment:
```suggestion
"    \\textit{\\textbf{\\textcolor{darkcandyapplered}{Feuermagienschule}}}\n"
```
Not like this?

Reply:
I my first draft i wrote "Feuermagieschule", but it was over two lines and it is not really necessary, because the symbols show the elements (fire, water, and so on) and the text next to it says clearly, that these four schools exist, so i neednt to repeat the word "Schule" four times in the descipting of the symbol.